Documentation/ApiOverview/Backend/JavaScript/Modules/MultiStepWizard.rst

```diff
@@ -1,40 +1,144 @@
-.. include:: /Includes.rst.txt
-.. index:: JavaScript (Backend); Multi-step wizard
-.. _modules-multistepwizard:
+..  include:: /Includes.rst.txt
+..  index:: JavaScript (Backend); Multi-step wizard
+..  _modules-multistepwizard:
 
 =================
 Multi-step wizard
 =================
 
-The JavaScript module :js:`MultiStepWizard` can be used to show a modal multi-step
-wizard with the following features:
-
-* Navigation to previous / next steps
-* Steps may have descriptive labels like "Start" or "Finish!"
-* Steps may require actions before becoming available.
-
-Code examples:
-
-.. code-block:: js
-
-   // Show/ hide the wizard
-   MultiStepWizard.show();
-   MultiStepWizard.dismiss();
-
-   // Add a slide to the wizard
-   MultiStepWizard.addSlide(
-       identifier,
-       stepTitle,
-       content,
-       severity,
-       progressBarTitle,
-       function() {
-       ...
-       }
-   );
-
-   // Lock/ unlock navigation buttons
-   MultiStepWizard.lockNextStep();
-   MultiStepWizard.unlockNextStep();
-   MultiStepWizard.lockPrevStep();
-   MultiStepWizard.unlockPrevStep();
+The JavaScript module :js:`MultiStepWizard` can be used to show a modal
+multi-step wizard with the following features:
+
+*   Navigation to previous / next steps
+*   Steps may have descriptive labels like "Start" or "Finish!"
+*   Steps may require actions before becoming available.
+
+..  _modules-multistepwizard-addslide:
+
+Add Slide
+=========
+
+You have to define at least one slide :javascript:`MultiStepWizard.addSlide()`.
+
+..  confval-menu::
+    :name: multi-step-wizard-settings
+    :display: table
+    :type:
+
+    ..  confval:: identifier
+        :name: multi-step-wizard-settings-identifier
+        :Required: true
+        :type: string
+
+        A unique identifier for the slide
+
+    ..  confval:: title
+        :name: multi-step-wizard-settings-title
+        :Required: true
+        :type: string
+
+        The title of the slide. Will be shown as header of the slide.
+
+    ..  confval:: content
+        :name: multi-step-wizard-settings-content
+        :Required: true
+        :type: string|JQuery|Element|DocumentFragment
+
+        The content of the slide. If `string` any HTML will be escaped. To
+        prevent that wrap your content into a jquery object:
+
+        ..  code-block:: js
+
+            $(`Your HTML content`);
```

Comment:
Nor sure JQuery is still recommended?

Reply:
Search for „jquery“ here: 

https://github.com/TYPO3/typo3/blob/main/Build/Sources/TypeScript/backend/multi-step-wizard.ts#L492

It will be imported, it is the return value of the component and all individual slides and it is at position two of the allowed data types